tests/all/typed_continuations.rs

```diff
@@ -1128,58 +1128,3 @@ mod traps {
         Ok(())
     }
 }
-
-mod misc {
-    use super::test_utils::*;
-    use wasmtime::*;
-
-    #[test]
-    pub fn continuation_revision_counter_wraparound() -> Result<()> {
-        let wat = r#"
-(module
-  (type $ft (func))
-  (type $ct (cont $ft))
-
-  (tag $yield)
-
-  (func $loop
-    (loop $loop
-      (suspend $yield)
-      (br $loop)
-    )
-  )
-  (elem declare func $loop)
-
-  ;; Loops 65536 times to overflow the 16 bit revision counter on the continuation reference.
-  (func (export "entry")
-    (local $k (ref $ct))
-    (local $i i32)
-    (local.set $k (cont.new $ct (ref.func $loop)))
-    (loop $go-again
-      (block $on-yield (result (ref $ct))
-        (resume $ct (tag $yield $on-yield) (local.get $k))
-        (unreachable)
-      )
-      (local.set $k)
-      (local.set $i (i32.add (i32.const 1) (local.get $i)))
-      (br_if $go-again (i32.lt_u (local.get $i) (i32.const 65536)))
-    )
-  )
-)
-"#;
-
-        let runner = Runner::new();
-        cfg_if::cfg_if! {
-            if #[cfg(feature = "unsafe_disable_continuation_linearity_check")] {
-                let result = runner.run_test::<()>(wat, &[])?;
-                assert_eq!(result, ())
-            } else {
-                let error = runner.run_test::<()>(wat, &[])
-                    .expect_err("expected an overflow");
-                assert!(error.root_cause().is::<Trap>());
-                assert_eq!(*error.downcast_ref::<Trap>().unwrap(), Trap::IntegerOverflow);
-            }
-        }
-        Ok(())
-    }
-}
```

Comment:
I'd like to preserve this test, albeit it is not feasible to run until we can seed the revision counter (e.g. via an embedder API or some configurable in the WasmFX config).